src/Toolchain/ToolchainProvider.ts

```diff
@@ -161,7 +159,7 @@ export class ToolchainProvider implements vscode.TreeDataProvider<ToolchainNode>
     });
   }
 
-  run(cfg: string) {
+  run(_cfg: string) {
```

Comment:
Could you let us know why this change is needed? Thanks in advance.

Reply:
This changes came from SonarLint. If this variable is not used in this function, SonarLint suggests removing this variable or adding `_` prefix before the word.

SonarLint warning:
> Remove the unused function parameter "cfg" or rename it to "_cfg" to make intention explicit.sonarlint(typescript:S1172)